tsconfig.json

```diff
@@ -1,30 +1,20 @@
 {
-  "compileOnSave": true,
-  "declaration": true,
-  "types": "index.d.ts",
   "compilerOptions": {
-    "incremental": true,
```

Comment:
Why did you remove all of that? I kinda liked it )

Reply:
when this is required, or useful? )

Reply:
incremental option with tsBuildInfoFile builds faster, target 2015 is to support older js versions, esModuleInterop allows to import without "* as". And why do you exclude __tests__ ?

Reply:
incremental + 
tests is not required for package publishing (less library package weight), and you execute them with ts-node

Reply:
incremental was there, i had change from top to bottom position in compilerOptions object(

Reply:
I got what you mean. Tests don't need tsconfig because they are executed with ts-node and they don't have to be built. Agree